test/routing/VersionRoutingRequestHandlerSpec.scala

```diff
@@ -79,7 +79,7 @@ class VersionRoutingRequestHandlerSpec extends UnitSpec with Inside with MockApp
     val httpConfiguration: HttpConfiguration = HttpConfiguration("context")
     private val errorHandler                 = mock[HttpErrorHandler]
     private val filters                      = mock[HttpFilters]
-    (filters.filters _).stubs().returns(Seq.empty)
+    (() => filters.filters).stubs().returns(Nil)
```

Comment:
There are several uses of Json.toJson(errorName) in this file which can be replaced with errorName.asJson